src/App.vue

```diff
@@ -1,282 +1,61 @@
-      background: -webkit-linear-gradient(top,  rgba(255,255,255,1) 0%,rgba(255,255,255,1) 65%,rgba(255,255,255,0) 100%); /* Chrome10-25,Safari5.1-6 */
-      background: linear-gradient(to top,  rgba(255,255,255,1) 0%,rgba(255,255,255,1) 65%,rgba(255,255,255,0) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
-      width: 100%;
-      left: 0;
-      top: -60px;
-      height: 140px;
-      z-index: -1;
-  }
-  .reload-button{
-    border-radius: 0.8em !important;
-    height: 45px;
-    font-size: 20px !important;
-    font-weight: 700 !important;
-    border: none !important;
-    box-shadow: none !important;
-    color: black !important;
-    box-shadow: 0px 0px 5px rgba(0,0,0,0.2) !important;
-  }
-  .gradient{
-    margin-right: -15px;
-    margin-left: -15px;
-  }
-  .header-wrapper{
-    display: flex;
-    align-items: center;
-    justify-content: center;
-  }
-
-@media screen and (min-width: 720px){
-  .info-button-wrapper{
-    justify-content: flex-end;
-  }
-  .dark-mode-button-wrapper{
-    justify-content: flex-start;
-  }
+html, body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', sans-serif;
+  background-color: white
 }
 
-  .header-buttons,
-  .header-buttons:focus{
-    background: none;
-    border: none;
-    outline: none;
-    margin-top: -10px;
-    font-size: 20px;
-  }
-
-  .dark-mode .header-buttons {
-    color: white;
-  }
-
-  .dark-mode button.close{
-    color: white;
-    text-shadow: none;
-    opacity: 0.75;
-  }
+.dark-mode, .dark-mode body, .dark-mode .modal-content {
+  background: #000814;
+  color: white;
+}
 
-  .header{
-    position: fixed;
-    width: 100%;
-    z-index: 10;
-    top: 0px;
-    padding: 0 15px;
-  }
+.dark-mode #reload-button::after, .dark-mode .header::before {
+  background: -moz-linear-gradient(top, rgba(0, 8, 20, 1) 0%, rgba(0, 8, 20, 1) 65%, rgba(0, 8, 20, 0) 100%); /* FF3.6-15 */
+  background: -webkit-linear-gradient(top, rgba(0, 8, 20, 1) 0%, rgba(0, 8, 20, 1) 65%, rgba(0, 8, 20, 0) 100%); /* Chrome10-25,Safari5.1-6 */
+  background: linear-gradient(to top, rgba(0, 8, 20, 1) 0%, rgba(0, 8, 20, 1) 65%, rgba(0, 8, 20, 0) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+}
 
-  .header::before{
-    content: '';
-    position: absolute;
-    width: 100%;
-    height: 80px;
-    margin-left: -15px;
-    background: -moz-linear-gradient(top,  rgba(255,255,255,1) 0%, rgba(255,255,255,1) 65%, rgba(255,255,255,0) 100%); /* FF3.6-15 */
-    background: -webkit-linear-gradient(top,  rgba(255,255,255,1) 0%,rgba(255,255,255,1) 65%,rgba(255,255,255,0) 100%); /* Chrome10-25,Safari5.1-6 */
-    background: linear-gradient(to top,  rgba(255,255,255,1) 0%,rgba(255,255,255,1) 65%,rgba(255,255,255,0) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
-    transform: rotate(180deg);
-  }
+#app {
+  min-width: 100%;
+  min-height: 100%;
+}
 
-  .content-container{
-    margin-top: 60px;
-  }
 
+.heading {
```

Comment:
Gibt es einen speziellen Grund, warum diese Klassen nicht in die Home.vue geschoben wurden?